index.html

```diff
@@ -1000,6 +1004,11 @@
 
 The base albedo $E_b$ which appears in equation [general_darkening_formula] represents the normal-incidence albedo of the entire base beneath the coat. This albedo can be approximated as the normal-incidence albedos of the individual slabs of the base, blended according to their mix weights.
 
+![](images/coat_darkening_0.0.png width=99%) ![](images/coat_darkening_0.5.png width=99%) ![](images/coat_darkening_1.png width=99%)
+<div class="shifted-caption">
+    ![Figure [fuzz]: A checkerboard **`coat_weight`**, varying the **`coat_darkening`** over 0, 0.5, 1 (default).](dummy)
```

Comment:
I find this particular illustration difficult to understand. The wood looks glossy everywhere, which I interpret as coated everywhere. If the intent is to show a checker of coated vs non coated parts, I'd suggest using a rough base layer.